lib/srv/regular/proxy.go

```diff
@@ -38,13 +38,13 @@ import (
 	"github.com/gravitational/teleport/lib/utils"
 
 	"github.com/gravitational/trace"
-	log "github.com/sirupsen/logrus"
```

Comment:
what's up with this refactoring?

Reply:
Moved to our new structured logging everywhere in `lib/srv`.

Reply:
how does renaming to logrus help?